FrontEnd/FrontEnd.Web/Services/IdentityService.cs

```diff
@@ -0,0 +1,20 @@
+using Identity.Public;
+
+namespace FrontEnd.Web.Services;
+
+public interface IIdentityService
+{
+    Task<IEnumerable<User>> GetAllAsync();
+}
+
+public class IdentityService(IHttpClientFactory httpClientFactory) : IIdentityService
```

Comment:
Maybe it would make more sense to use [Typed clients](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-8.0#typed-clients)?